tests/test_pyxtal.py

```diff
@@ -27,7 +27,12 @@ def test_args_raised(self):
             stk.build.pyxtal(1, species=["Fe", "Cr"], num_ions=[1])
 
         try:
-            stk.build.pyxtal([193, 194], ["Mg"], num_ions=[1], allow_exceptions=True)
+            with self.assertWarnsRegex(
+                    UserWarning,
+                    regex="Groups [193, 194] could not be generated with stoichiometry Mg1!",
```

Comment:
```suggestion
                    "Groups [193, 194] could not be generated with stoichiometry Mg1!",
```

The name of argument seems to be incorrectly documented in cpython.